torch/_inductor/ir.py

```diff
@@ -3219,9 +3219,19 @@ def __init__(
         )
         self.use_cpp_op_schema = False
 
-        if getattr(torch.ops.aten, kernel.__name__, None) is kernel:
+        def is_aten_op(kernel):
+            obj = torch.ops.aten
+
+            for part in kernel.__name__.split("."):
+                obj = getattr(obj, part, None)
+                if obj is None:
+                    return False
+
+            return obj is kernel
```

Comment:
Is it good enough to check `kernel.namespace == 'aten'`, considering it is ready resolved as OpOverload?